HierarchicalAgenticRAG/tools/worker_agent_tools.yaml

```diff
@@ -0,0 +1,5 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+duckduckgo_search:
+  callable_api: ddg-search
```

Comment:
duckduckgo is just a placeholder for RagRetriever right? 

Does that mean we still need to have DocIndexer to complete the PR eventually?
https://github.com/opea-project/GenAIExamples/issues/415

Reply:
Yes, we will update this example in v1.0 or later with retrieval megaservice as mentioned in the roadmap section in readme.